src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -3245,10 +3137,10 @@ namespace HVACMultiSpeedHeatPump {
 
         OutsideDryBulbTemp = state.dataEnvrn->OutDryBulbTemp;
 
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:3248]:(style),[constVariable],Variable 'MSHeatPump' can be declared as reference to const